src/migrations/user-store/file-name-migration.ts

```diff
@@ -0,0 +1,17 @@
+import fse from "fs-extra";
+import { app, remote } from "electron";
+import path from "path";
+
+export async function fileNameMigration() {
+  const userDataPath = (app || remote.app).getPath("userData");
+  const configJsonPath = path.join(userDataPath, "config.json");
+  const lensUserStoreJsonPath = path.join(userDataPath, "lens-user-store.json");
```

Comment:
since `configName` is now set can you use `userStore.path()` here? Or at least some global that defines `"lens-user-store"`?

Reply:
I don't think that we should use importing `userStore` here.

Reply:
Though looking at this code, it is rather flaky so I'll make some changes and ask for re-review